ci/pod/docker-compose.yml

```diff
@@ -402,11 +402,14 @@ services:
     restart: unless-stopped
     ports:
       - 8181:8181
-    command: run -s /example.rego /data.json
+    command: run -s /example.rego /example2.rego /data.json
```

Comment:
Better to give a reason name instead of exampleX

Reply:
It is only used for testing purposes and its function is to return all the data in the input, so I have changed it to `debug`.